### ISSUE TITLE
1.3.0.6 doesnt work anymore, 1.4.0.2 segmentation fault - Raspberry PI

### ISSUE THREAD

**Issue by sigi77777** [hpool-dev/chia-miner#537]
What to do????
This is very bad support, already 7 days passed since release and still no working new executable for rapsberry pi.

**Comment by almando** [comment]
I have the same problem. Using Raspberry Pi 4, 8GB RAM on Ubuntu Linux

**Comment by tokenhoki** [comment]
i have same problem with 1.4.02

**Comment by kamil234** [comment]
I reported this issue long time ago and still no update...
https://github.com/hpool-dev/chia-miner/issues/490

**Comment by sigi77777** [comment]
> I reported this issue long time ago and still no update...
> #490

the issue is even bigger now... Old version doesnt start. Screen with github relaeases appear in a web browser and the app exits...

**Comment by gddrx** [comment]
same problem -__-

**Comment by N0Klu3** [comment]
Good flippen Job Hpool! 
Fix the issues before you ban a version when there are many people saying it isnt working... 

Cant wait for actual pooling to come out so it doesnt take me over a day to get my withdrawals and to put up with you guys having to make your own hacky way of doing things...

Fix the arm version or allow the old client!

**Comment by jacek900** [comment]
I fully agree. Can't ban the old one and don't even try to comment on everybody's issues.

**Comment by vespassassina** [comment]
Same problem, you broke both Raspberry versions (both armv7 and arch64 are broken) and blocked the one that was still working (the old 1.3.6 armv7).
 
It is ridiculous and totally bad management and bad coding.

**Comment by dreamboostme** [comment]
i have same problem

**Comment by manfye** [comment]
+1

**Comment by matheust3** [comment]
+1

**Comment by jorgefuertes** [comment]
Or release the source, so anyone else can fix it for his own architecture.

**Comment by StormYudi** [comment]
Same issues, I have to turn back to x86 to keep the miner work, which is I don't want to see.

**Comment by xicexice** [comment]
垃圾黑心矿池偷我们算力

**Comment by arontsang** [comment]
+1

**Comment by trying2016** [comment]
We tested it under Linux raspberrypi 5.10.17-v7l+ #1414 SMP Fri Apr 30 13:20:47 BST 2021 armv7l GNU/Linux

**Comment by jacek900** [comment]
Re-test it on Ubuntu 21.04 please

**Comment by arontsang** [comment]
Tested using docker image

ubuntu:rolling aarch64

Exact same error.

**Comment by arontsang** [comment]
> Re-test it on Ubuntu 21.04 please

Tested using docker image

ubuntu:21.04 aarch64
ubuntu:latest aarch64

Exact same error.

@trying2016

**Comment by arontsang** [comment]
@jacek900 I suggest you test your code on AWS graviton servers.

**Comment by N0Klu3** [comment]
> We tested it under Linux raspberrypi 5.10.17-v7l+ #1414 SMP Fri Apr 30 13:20:47 BST 2021 armv7l GNU/Linux

Funny how I'm also using Ubuntu MATE 21.04, on RPI4 and I am having issues, with kernel 5.11
Also are you saying ALL the issues being raised we're all wrong? Because you tested?

**Comment by sigi77777** [comment]
> We tested it under Linux raspberrypi 5.10.17-v7l+ #1414 SMP Fri Apr 30 13:20:47 BST 2021 armv7l GNU/Linux

Can anyone confirm that it works?

**Comment by arontsang** [comment]
@sigi77777 No one can confirm, because it doesn't work.

I can confirm it doesn't work.

**Comment by kamil234** [comment]
@N0Klu3 there's no need to 'attack' dev for simply saying they tested it. But yes, the issue is still present (im using Ubuntu 21.04). 

Linux ubuntu 5.4.0-1035-raspi #38-Ubuntu SMP PREEMPT Tue Apr 20 21:40:35 UTC 2021 armv7l armv7l armv7l GNU/Linux

Every time get SEG FAULT

**Comment by N0Klu3** [comment]
> @N0Klu3 there's no need to 'attack' dev for simply saying they tested it. But yes, the issue is still present (im using Ubuntu 21.04).
> 
> Linux ubuntu 5.4.0-1035-raspi #38-Ubuntu SMP PREEMPT Tue Apr 20 21:40:35 UTC 2021 armv7l armv7l armv7l GNU/Linux
> 
> Every time get SEG FAULT

I’m not attacking him. I am annoyed how they can block a working miner when there are MANY reports of this version not working and now many of us are unable to mine!

I have now waisted a few days of no mining results because they blocked the old version without first fixing the new one. This is my annoyance. And honestly it’s pretty poor on their part!

**Comment by Cattigrew** [comment]
Same happening on the ubuntu 20.04 running on a docker

**Comment by albertoanta** [comment]
Same problem on rpi4 1gb ubuntu 21.04

**Comment by kamil234** [comment]
If you guys cant switch to Windows or dont want to wait any longer for fix, reimage your raspberryPi with Raspbian (instead of ubuntu, or whatever else you're running) and you can run 1.4.0.2 without any issue. I just did it, and it works.

**Comment by jacek900** [comment]
I heard there is a "not enough resources" (or similar) error when you have more than 6 hard drives and use Windows on PI - anyone to confirm or reject?

**Comment by Hexagonale** [comment]
Best solution for now is to run arm32v7 docker image. Install docker and use this [image](https://hub.docker.com/r/arm32v7/ubuntu/). Mount your volumes using `--mount type=bind,source=/mnt/1,target=/mnt/1` parameter. I've created basic script to run it.

For installing:
`sudo apt install docker.io`
`sudo docker pull docker pull arm32v7/ubuntu`

For running:
[this gist](https://gist.github.com/Hexagonale/4c050b0144bc895c6c54e29cb72bfcb7)
Remember to change path to your hpool miner (or just downlod it from inside).
Then you just run miner inside and voila.

**Comment by CoMMyz** [comment]
Indeed it works with Raspbian but not Ubuntu image for RPi. Some library is probably missing. Have in mind that Raspbian is 32bit but Ubuntu is running 64bit

**Comment by Hexagonale** [comment]
I'm running it on 64bit ubuntu. Image is 32bit and miner needs to be for arm not aarch64 for this to work.

**Comment by Cattigrew** [comment]
> Best solution for now is to run arm32v7 docker image. Install docker and use this [image](https://hub.docker.com/r/arm32v7/ubuntu/). Mount your volumes using `--mount type=bind,source=/mnt/1,target=/mnt/1` parameter. I've created basic script to run it.
> 
> For installing:
> `sudo apt install docker.io`
> `sudo docker pull docker pull arm32v7/ubuntu`
> 
> For running:
> [this gist](https://gist.github.com/Hexagonale/4c050b0144bc895c6c54e29cb72bfcb7)
> Remember to change path to your hpool miner (or just downlod it from inside).
> Then you just run miner inside and voila.

Thanks it works now
